login/SingIn.js

```diff
@@ -0,0 +1,26 @@
+const UserResponse = require("./UserResponse.js");
+const UserModel = require("./Models/UserModel.js");
+
+module.exports.SingIn = function SingIn (loginRequest,request,response)
+{
+    UserModel.User.findOne({raw:true,where: {login: loginRequest.login,password:loginRequest.password}})
+        .then(User=>{
+            if(!User){
+                console.log("Данные введены неправильно");
+                return;
+            }
+            else {
+                let userTables = JSON.parse(JSON.stringify(User));
```

Comment:
You could probably abstract this db logic away from the SignIn.js, it doesn't need to know about how you create and populate the UserResponse object, it would be better to just have something like let userResponse = UserService.GetUser(id) or something similar.

Reply:
Ok,i will try it today